qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/commons/GenerateMissingJavaMemberParams.java

```diff
@@ -98,4 +98,28 @@ public void setProjectUri(String projectUri) {
 		this.projectUri = projectUri;
 	}
 
+	@Override
+	public String toString() {
+		StringBuilder buf = new StringBuilder("GenereateMissingJavaMemberParams [\n");
```

Comment:
Why not using ToStringBuilder?

Reply:
Thanks! Didn't know about this method.

Reply:
you did it here https://github.com/redhat-developer/quarkus-ls/pull/686/files#diff-70b7e08819155fab891313cee90b7612e06e38ebfda05864c4ed37df00621807R259 :)